### DISCUSSION TITLE
How to use Graphic Walker in ObservableHQ?

### DISCUSSION THREAD

**Discussion by lfkpoa** [Kanaries/graphic-walker#71] [Q&A]
Congratulations!

Graphic Walker is really great!
I've been trying to use it in ObservableHQ notebooks, but I couldn't manage to make it work, since it uses React.
I've tried to use the following notebooks to load the React component:

https://observablehq.com/@j-f1/react
https://observablehq.com/@elisk/styled

This is my attempt:
https://observablehq.com/d/ada21f9250cdec47

Anybody could help with this?
Making it available in ObservableHQ would be awesome.

Thank you.

**Comment by declann** [comment]
+1 I think this would be awesome.
